src/implementations/twig/components/file/file.html.twig

```diff
@@ -128,7 +133,12 @@
       </div>
     {% endif %}
     {% if _title is not empty %}
-      <div class="ecl-file__title">
+      <div class="ecl-file__title"
```

Comment:
if I'm not wrong, the title id is used in any case for the download button. So the id should be there even without translations